src/layer/tile/GridLayer.js

```diff
@@ -444,7 +445,8 @@ L.GridLayer = L.Layer.extend({
 
 	_getTiledPixelBounds: function (center) {
 		var map = this._map,
-		    scale = map.getZoomScale(map.getZoom(), this._tileZoom),
+		    mapZoom = map._animatingZoom ? Math.max(map._animateToZoom, map.getZoom()) : map.getZoom(),
+		    scale = map.getZoomScale(mapZoom, this._tileZoom),
```

Comment:
Sorry to insist a bit, but as per my understanding we don't need scale: if the map size is 600px at zoom 10 it will still be 600px at zoom 11 or 17 or 17.358, isn't it? Or am I missing something?